### PR TITLE
Add requests to create namespace modal

### PR DESCRIPTION
**Pull request by y-kkamil**
**Description**

Changes proposed in this pull request:

- Add gql requests to create namespace modal

**Related issue(s)**
See #1296

### CODE REVIEW

File: compass/src/shared/components/ModalWithForm/ModalWithForm.component.js [hunk 1]
```diff
@@ -66,7 +67,7 @@ const ModalWithForm = ({
       <Modal
         show={isOpen}
         actions={
-          <>
+          <React.Fragment>
```
Comment (sjanota) [inline compass/src/shared/components/ModalWithForm/ModalWithForm.component.js]: out of curiosity: why? `<>` is so short and nice :)

File: core-ui/.env [hunk 2]
```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```
Comment (sjanota) [inline core-ui/.env]: Why?

File: core-ui/src/contexts/notifications/index.js [hunk 3]
```diff
@@ -23,17 +27,19 @@ export const NotificationProvider = ({
           visibilityTime = defaultVisibilityTime,
         ) {
           setState({ isOpen: true, notificationProps });
-          setTimeout(() => {
-            setState({ isOpen: false });
-          }, visibilityTime);
+          if (notificationProps.autoClose) {
+            setTimeout(() => {
+              setState({ isOpen: false });
```
Comment (sjanota) [inline core-ui/src/contexts/notifications/index.js]:
```suggestion
              closeNotification();
```

File: compass/src/shared/components/ModalWithForm/ModalWithForm.component.js (hunk 1 above)
Comment (y-kkamil) [reply compass/src/shared/components/ModalWithForm/ModalWithForm.component.js]: it came from: https://github.com/kyma-project/console/pull/1300/files/9430a589205408872c96386714df67611febf61b#diff-98574fa08ec01aa62a6981efd8a52684R74 i believe it's fixed there already ;)

File: core-ui/src/components/CreateNamespaceForm/test/CreateNamespaceForm.test.js [hunk 4]
```diff
@@ -61,4 +72,128 @@ describe('CreateNamespaceForm', () => {
...
+    });
+
+    expect(gqlMock[0].result).toHaveBeenCalled();
+    expect(gqlMock[1].result).toHaveBeenCalled();
+    expect(gqlMock[2].result).toHaveBeenCalled();
+    expect(onCompleted).toHaveBeenCalled();
+    expect(onError).not.toHaveBeenCalled();
+  });
+
+  it('Shows warning, when Namespace was created, but Limit Range or Resource quota creation failed ', async () => {
```
Comment (sjanota) [inline core-ui/src/components/CreateNamespaceForm/test/CreateNamespaceForm.test.js]: How about error during namespace creation?

Comment (y-kkamil) [reply core-ui/src/components/CreateNamespaceForm/test/CreateNamespaceForm.test.js]: Added

**Pull request merged**